core/modules/contextual/js/contextual.js

```diff
@@ -16,30 +16,107 @@ Backdrop.behaviors.contextualLinks = {
       var $wrapper = $(this);
       var $region = $wrapper.closest('.contextual-links-region');
       var $links = $wrapper.find('ul.contextual-links');
-      var $trigger = $('<a class="contextual-links-trigger" href="#" />').text(Backdrop.t('Configure')).on('click',
+      var $trigger = $('<a class="contextual-links-trigger" href="#" />').text(Backdrop.t('Configure')).click(
         function () {
           $links.stop(true, true).slideToggle(100);
           $wrapper.toggleClass('contextual-links-active');
           return false;
         }
       );
       // Attach hover behavior to trigger and ul.contextual-links.
-      $trigger.add($links).on('hover',
+      $trigger.add($links).hover(
         function () { $region.addClass('contextual-links-region-active'); },
         function () { $region.removeClass('contextual-links-region-active'); }
       );
       // Hide the contextual links when user clicks a link or rolls out of the .contextual-links-region.
-      $region.on('mouseleave click', Backdrop.contextualLinks.mouseleave);
-      $region.on('hover',
+      $region.bind('mouseleave click', Backdrop.contextualLinks.mouseleave);
+      $region.hover(
         function() { $trigger.addClass('contextual-links-trigger-active'); },
         function() { $trigger.removeClass('contextual-links-trigger-active'); }
       );
       // Prepend the trigger.
       $wrapper.prepend($trigger);
-
-      // Prevent child contextual links overlapping their parent ones.
-      $wrapper.siblings().find('.contextual-links-wrapper').eq(0).css('margin-top', '20px');
     });
+
+    /**
+     * Adjusts trigger positions in contextual links to avoid overlaps.
+     */
+    function adjustContextualLinks() {
+      // Get all wrappers anywhere on the page and some info about each.
+      var allWrappers = [];
+      $('.contextual-links-wrapper', context).each(function() {
+        allWrappers.push({
+          'wrapper': $(this),
+          'regionOffsetBottom': $(this).parent().offset().top + $(this).parent().height(),
+          'hshift': 0,
+          'vshift': 0
```

Comment:
To avoid these spelling notices, maybe change these variables to `xShift` and `yShift` or `hShift` and `vShift`

Reply:
Will do.